presto-docs/build

```diff
@@ -6,7 +6,7 @@ cd "${BASH_SOURCE%/*}"
 
 test -t 1 && OPTS='-it' || OPTS=''
 
-SPHINX_IMAGE=${SPHINX_IMAGE:-prestodev/sphinx:2}
+SPHINX_IMAGE=${SPHINX_IMAGE:-prestodev/sphinx:latest}
```

Comment:
We should continue to depend on an exact version so that builds are deterministic.